src/ui/Views/Auth/AccessDenied.cshtml

```diff
@@ -1,20 +0,0 @@
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <h1 class="heading-xlarge">
-      We don’t know which organisation you’re part of
-    </h1>
-
-    <p>You have successfully signed in with your DfE account but we don’t recognise the email address you’ve used. This might have happened if you were forwarded the original invitation email or if you’ve recently updated your email address.</p>
-    <p>We need more information from you before we can give you access to your organisation’s courses.</p>
```

Comment:
We'll have to figure out another way of displaying this info before merging this I think.  I agree it shouldn't be the blanket 401 text, that was just a temporary hack.

Reply:
Agreed we can't lose the functionality of showing this copy when a user is authenticated but unauthorised.

Reply:
Okay, so we have replaced this specific text with `This page is restricted to certain users only.` but if we want different copy for each use case then we need to make this text dynamic so when we raise this exception we pass in the message 👍